srcpkgs/matterhorn/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'matterhorn'
+pkgname=matterhorn
+version=50200.7.0
+revision=1
+makedepends="cabal-install ghc zlib-devel ncurses-devel"
+depends="gmp zlib"
```

Comment:
You already depend on zlib in makedepends, gmp should be picked up automatically too if something links against it.

Reply:
True, I removed it